DOCSP-40417: Add X509 troubleshooting steps (#213)
mongodb/docs-csharp: source/fundamentals/authentication.txt

         You can alternatively pass ``null`` as the parameter to prompt the MongoDB
         server to infer the username based on your ``X.509`` certificate.

.. note::

>    If you are using Windows, the driver may be unable to locate an ``X.509``
>    authentication certificate created in memory. To learn more about a potential solution to
>    this issue, see the :ref:`Check Ephemeral Key Usage <csharp-troubleshooting-connection-X509>`
>    section of the Connection Troubleshooting guide.

Reviewer: ```suggestion
   If you are using Windows, the driver might be unable to locate an ``X.509``
   authentication certificate created in memory. To learn more about a potential solution to
   this issue, see the :ref:`csharp-troubleshooting-connection-X509`
   section of the Connection Troubleshooting guide.
```

>    There is a known bug on Windows where the driver may be unable to locate an
>    ``X.509`` authentication certificate created in memory. To learn more about troubleshooting
>    this issue, see the :ref:`Check Ephemeral Key Usage <csharp-troubleshooting-connection-X509>`
>    section of the Connection Troubleshooting guide.

Reviewer: ```suggestion
   If you are using Windows, the driver may be unable to locate an
   ``X.509`` authentication certificate created in memory. To learn about a potential solution to
   this issue, see the :ref:`Check Ephemeral Key Usage <csharp-troubleshooting-connection-X509>`
   section of the Connection Troubleshooting guide.
```
